content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -105,16 +162,38 @@ To encrypt the data:
          - secrets
        providers:
          - kms:
-             name: myKmsPlugin
+             name: myKmsPluginFoo
+             endpoint: unix:///tmp/socketfile.sock
+             cachesize: 100
+             timeout: 3s
+         - kms:
+             name: myKmsPluginBar
              endpoint: unix:///tmp/socketfile.sock
              cachesize: 100
              timeout: 3s
-         - identity: {}
    ```
 
-1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to
-   the location of the configuration file.
-1. Restart your API server.
+### KMS v2 {#encrypting-your-data-with-the-kms-provider-kms-v2}
+
+   ```yaml
+   apiVersion: apiserver.config.k8s.io/v1
+   kind: EncryptionConfiguration
+   resources:
+     - resources:
+         - secrets
+       providers:
+         - kms:
+             apiVersion: v2
+             name: myKmsPluginFoo
+             endpoint: unix:///tmp/socketfile.sock
+             cachesize: 100
+             timeout: 3s
+         - kms:
+             name: myKmsPluginBar
+             endpoint: unix:///tmp/socketfile.sock
+             cachesize: 100
+             timeout: 3s
+   ```
 
 For details about the `EncryptionConfiguration` format, please check the
```

Comment:
```suggestion
Until the steps defined in `Ensuring all secrets are encrypted` are performed, the `providers` list should end with the `identity: {}` provider to allow unencrypted data to be read.  Once all resources are encrypted, the `identity` provider should be removed to prevent the API server from honoring unencrypted data.

For details about the `EncryptionConfiguration` format, please check the
```